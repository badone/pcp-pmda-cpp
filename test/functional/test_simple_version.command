pmdasimple --version | grep -v '^PCP version'
