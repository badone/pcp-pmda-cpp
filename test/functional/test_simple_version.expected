
simple PMDA
PMDA interface version 6
PCP version 3.9.1 (30901)

